src/Console/Command/CommandHelp.php

```diff
@@ -279,16 +337,6 @@ function (FixerInterface $a, FixerInterface $b) {
                 $description = '[n/a]';
             }
 
-            $attributes = array();
-
-            if ($fixer->isRisky()) {
-                $attributes[] = 'risky';
-            }
-
-            if ($fixer instanceof ConfigurableFixerInterface) {
-                $attributes[] = 'configurable';
-            }
-
             $description = wordwrap($description, 72, "\n   | ");
```

Comment:
what is the difference between this function and newly introduced `wordwrap` method ?